From e65a31c9ef6fd295910bef29af675127ed0639ef Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Fri, 7 Dec 2018 19:30:03 +0530
Subject: [PATCH 047/235] staging: fsl_ppfe/eth: remove unused code

- remove gemac-bus-id related code that is unused.
- remove unused prototype gemac_set_mdc_div.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl_ppfe/include/pfe/pfe.h      | 1 -
 drivers/staging/fsl_ppfe/pfe_eth.h              | 1 -
 drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c | 7 -------
 3 files changed, 9 deletions(-)

diff --git a/drivers/staging/fsl_ppfe/include/pfe/pfe.h b/drivers/staging/fsl_ppfe/include/pfe/pfe.h
index 79eeafa60eb6..c5b5ce31a122 100644
--- a/drivers/staging/fsl_ppfe/include/pfe/pfe.h
+++ b/drivers/staging/fsl_ppfe/include/pfe/pfe.h
@@ -268,7 +268,6 @@ enum mac_loop {LB_NONE, LB_EXT, LB_LOCAL};
 void gemac_init(void *base, void *config);
 void gemac_disable_rx_checksum_offload(void *base);
 void gemac_enable_rx_checksum_offload(void *base);
-void gemac_set_mdc_div(void *base, int mdc_div);
 void gemac_set_speed(void *base, enum mac_speed gem_speed);
 void gemac_set_duplex(void *base, int duplex);
 void gemac_set_mode(void *base, int mode);
diff --git a/drivers/staging/fsl_ppfe/pfe_eth.h b/drivers/staging/fsl_ppfe/pfe_eth.h
index 8153b9523c6d..787fbeb3ea93 100644
--- a/drivers/staging/fsl_ppfe/pfe_eth.h
+++ b/drivers/staging/fsl_ppfe/pfe_eth.h
@@ -41,7 +41,6 @@ struct ls1012a_eth_platform_data {
 	u32 mii_config;
 	u32 phy_flags;
 	u32 gem_id;
-	u32 bus_id;
 	u32 phy_id;
 	u32 mdio_muxval;
 	u8 mac_addr[ETH_ALEN];
diff --git a/drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c b/drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c
index 7fc67872adb2..d3ed6eca2069 100644
--- a/drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c
+++ b/drivers/staging/fsl_ppfe/pfe_ls1012a_platform.c
@@ -55,13 +55,6 @@ static int pfe_get_gemac_if_properties(struct device_node *parent, int port, int
 		       ETH_ALEN);
 	}
 
-	addr = of_get_property(gem, "fsl,gemac-bus-id", &size);
-	if (!addr)
-		pr_err("%s:%d Invalid gemac-bus-id....\n", __func__,
-		       __LINE__);
-	else
-		pdata->ls1012a_eth_pdata[port].bus_id = be32_to_cpup(addr);
-
 	phy_node = of_parse_phandle(gem, "phy-handle", 0);
 	pdata->ls1012a_eth_pdata[port].phy_node = phy_node;
 	if (phy_node) {
-- 
2.17.1

